File: Step1/Logical-Thinking/Pattern8/Solution.java
public class Main
{
    static void star(int n){
        for(int i = n; i>0; --i){
            for(int j = 0; j<=(n-i-1); ++j){
                System.out.print(" ");
            }
            for(int k = 0; k<(2*i-1); ++k){
                System.out.print("*");
            }
           System.out.println();
        }
    }
	public static void main(String[] args) {
		star(5);
	}
}

File: Step1/Logical-Thinking/Pattern8/Solution.cpp
/* SAMPLE PATTERN
*****
 ***
  *
  */
#include <iostream>
using namespace std;
void star(int n){
    for(int i = n; i>0; --i){
            for(int j = 0; j<=(n-i); ++j){
                cout<<" ";
            }
            for(int k = 0; k<(2*i-1); ++k){
                cout<<"*";
            }
            cout<<endl;
        }
}
int main()
{
    star(5);

    return 0;
}
